static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -102,6 +102,7 @@
         .content__head--mobile {
             position: relative;
             height: 100vh;
+            height: -webkit-calc(100vh - 48px); // To compensate for mobile Safari viewport height issue.
```

Comment:
So will this appear broken in Blink? Not sure if there's an alternative, can't think of one